### PR TITLE
Logging and updates

### PR DESCRIPTION
**Pull request by majst01**
This addresses #5 and #4. 

compiles and removed all fmt.Printf, tested with enhanced cli.

Still requires review and usage thought.

### CODE REVIEW

File: internal/redfish/redfish.go [hunk 1]
```diff
@@ -90,7 +94,7 @@ func (c *APIClient) BoardInfo() (*api.Board, error) {
 func (c *APIClient) MachineUUID() (string, error) {
 	systems, err := c.Service.Systems()
 	if err != nil {
-		log.Printf("ignored system query err:%s\n", err.Error())
+		c.log.Errorw("ignore system query", "error", err.Error())
 		return "", err
```
Comment (kolsa) [inline internal/redfish/redfish.go:98]: May you also remove the `return "", err` line and therewith rather use `c.log.Warnw`?
Although not critical yet there are chances that a valid system will be returned alongside an error that corresponds to another one

Comment (majst01) [reply internal/redfish/redfish.go:98]: But if UUID ist not accessible, this is considered fatal right ? Unsure though

Comment (kolsa) [reply internal/redfish/redfish.go:98]: Ok, that's right I think. Then we should proper update the error message

Comment (majst01) [reply internal/redfish/redfish.go:98]: ok error message now bit more explicit.

Comment (kolsa) [reply internal/redfish/redfish.go:98]: I mean we should not log `ignore system query` since we do not ignore it here

Comment (majst01) [reply internal/redfish/redfish.go:98]: Of course, stupid me. Now fixed.

**Comment by majst01** [comment]
had to move to pkg/logger because otherwise consumers cant consumer :-)

**Review by kolsa — APPROVED** [review]

**Review by Gerrit91 — APPROVED** [review]

**Pull request merged**